### PR TITLE
JTB-3: commit

### PR DESCRIPTION
**Pull request by Soqoor**

### CODE REVIEW

**Comment by Soqoor** [comment]
1

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # test-repo
 For test only
 For test only
+New row
```
Comment (rukos17) [inline README.md:4]: Whats this?

**Review by rukos17 — CHANGES_REQUESTED** [review]
Need to fix

**Review by rukos17 — COMMENTED** [review]
Ok for now

**Review by rukos17 — APPROVED** [review]
Ok approve

**Review by rukos17 — APPROVED** [review]
Sure approve

**Comment by rukos17** [comment]
Just comment

**Review by rukos17 — CHANGES_REQUESTED** [review]
Wait!

**Review by rukos17 — APPROVED** [review]
app ok

File: README.md (hunk 1 above)
Comment (rukos17) [inline README.md:4]: New comment on row

**Review by rukos17 — COMMENTED** [review]
rewiew with comment

**Review by rukos17 — COMMENTED** [review]
Commented text

**Review by rukos17 — CHANGES_REQUESTED** [review]
Need to fix

**Review by rukos17 — APPROVED** [review]

**Comment by Soqoor** [comment]
123qwe

**Pull request closed without merging**